content/en/docs/concepts/services-networking/service.md

```diff
@@ -983,20 +983,25 @@ The `.spec.trafficDistribution` field provides another way to influence traffic
 routing within a Kubernetes Service. While traffic policies focus on strict
 semantic guarantees, traffic distribution allows you to express _preferences_
 (such as routing to topologically closer endpoints). This can help optimize for
-performance, cost, or reliability. This optional field can be used if you have
-enabled the `ServiceTrafficDistribution` [feature
-gate](/docs/reference/command-line-tools-reference/feature-gates/) for your
-cluster and all of its nodes. In Kubernetes {{< skew currentVersion >}}, the
+performance, cost, or reliability. In Kubernetes {{< skew currentVersion >}}, the
 following field value is supported: 
```

Comment:
This line might need to change now there are other supported values.

Reply:
The "Update for PreferSameTrafficDistribution" commit adds the new values, but it adds it as a separate paragraph below this one since they're still Alpha:

```
+{{< feature-state feature_gate_name="PreferSameTrafficDistribution" >}}
+
+Two additional values are available when the `PreferSameTrafficDistribution`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is
+enabled:

...
```